v63003/inref/gtm8680.m

```diff
@@ -17,26 +17,22 @@
 	else  do ^job("mprocesseschild^gtm8680",100,"""""")
 	set max=100000
 	set maxit=0
-	set maxit2=0
 	set minit=0
-	set minit2=0
-	set ^its($increment(^i))=0
+	set its($increment(it))=0
 	set start=$h,cur=start,prev=start,previ=0,t=$$^difftime(cur,start)
-	for i=1:1 do  quit:(t>15)
+	for i=1:1 do  quit:(t>30)
```

Comment:
Why does this change show up in this pull request? All following changes fall in the same category. Some mixup.